apps/funding-service/hardhat.config.ts

```diff
@@ -1,2 +1,14 @@
 import "@nomiclabs/hardhat-ethers";
 import "@nomicfoundation/hardhat-chai-matchers";
+// import { HardhatUserConfig } from "hardhat/types";
+
+// const config: HardhatUserConfig = {
+//   networks: {
+//     hardhat: {
+//       // compatibility with hopr pluto hardhat
+//       chainId: 1,
+//     },
+//   },
+// };
+
+// export default config;
```

Comment:
@ItsLuized this can be needed to test funding service docker now that the hardhat chain id is equal to "1" in utils

Reply:
not going to be handled here